chart/flux/README.md

```diff
@@ -58,7 +58,7 @@ At startup Flux generates a SSH key and logs the public key.
 Find the SSH public key with:
 
 ```bash
-kubectl -n flux logs deployment/flux | grep identity.pub
+kubectl -n flux logs deployment/flux | grep identity.pub | cut -d '"' -f2
```

Comment:
It'd be nicer to do this with fluxctl if we can -- but perhaps in this context you're trying to avoid asking people to download a binary. Could we do it with a `kubectl run` or similar instead? (It's not necessary to change it here, we can improve it later)

Reply:
This is in the helm chart, where folks would need to have downloaded and set up `fluxctl` already. I'm happy to go with your guidance here. I just noticed the `grep identity.pub` in a few places and thought I'd make the calls all consistent.

For the hello-world type tutorial I'd probably default to using this as well, but mention the use of `fluxctl identity` as well.

How would you use `kubectl run` here?

Reply:
`kubectl run --image weaveworks/flux -- identity` or something like that -- a kind of in-cluster `fluxctl`.
(In case it's not clear, we'd have to do a big of engineering to get it to work that way, first!)